app/src/main/java/com/example/movopfy/features/title/presentation/TitleScreen.kt

```diff
@@ -0,0 +1,94 @@
+package com.example.movopfy.features.title.presentation
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import com.example.movopfy.R
+import com.example.movopfy.main.ui.theme.BackgroundMain
+import com.example.movopfy.main.ui.theme.Dimensions
+import com.example.movopfy.main.ui.theme.LocalDim
+import com.example.movopfy.main.ui.theme.TextMain
+import com.example.movopfy.uiComponents.ProgressBarLoading
+import org.koin.androidx.compose.koinViewModel
+
+@Composable
+fun TitleScreen(viewModel: TitleViewModel = koinViewModel(), id: Int) {
+
+    val uiState by viewModel.uiState.collectAsState()
+
+    viewModel.getTitleById(id = id)
+
+    val dimensions = LocalDim.current
+
+    Column(
+        modifier = Modifier
+            .background(color = BackgroundMain)
+            .fillMaxSize()
+    ) {
+        when (val state = uiState) {
+            is TitleViewModel.TitleUiState.Loading -> {
+                ProgressBarLoading()
+            }
+
+            is TitleViewModel.TitleUiState.Loaded -> {
+
+                LazyColumn {
+                    item {
+                        AsyncImage(
+                            model = "https://www.anilibria.tv${state.title?.anilibriaPosters?.original?.url}",
```

Comment:
maybe you can calculate it not on a ui level and just prove `url` here?

Reply:
but where if I have to get title from view model?